db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql

```diff
@@ -0,0 +1,99 @@
+-- +migrate Up
```

Comment:
Original import script https://developers.stellar.org/platforms/stellar-disbursement-platform/admin-guide/single-tenant-to-multi-tenant-migration#importing-your-data